.eslintrc.js

```diff
@@ -21,6 +21,24 @@ module.exports = {
 				'md/remark': [ 'off' ],
 			},
 		},
+		{
+			// This lints the codeblocks marked as `javascritp`, `js`, `cjs` or `ejs`, all valid aliases
```

Comment:
Should we also do TypeScript while we're at it? I only see around 20 references to `` ```ts `` when I search the codebase so it wouldn't add too much more.

```suggestion
			// This lints the codeblocks marked as `javascript`, `js`, `cjs` or `ejs`, all valid aliases
```

Reply:
Good point! I'll address it in a separate PR, as I need to investigate if remark has support for it